c10/core/TensorImpl.cpp

```diff
@@ -127,7 +127,7 @@ TensorImpl::TensorImpl(
     const caffe2::TypeMeta data_type,
     c10::optional<c10::Device> device_opt)
     // NOLINTNEXTLINE(performance-move-const-arg)
-    : TensorImpl({}, key_set, data_type, std::move(device_opt)) {}
+    : TensorImpl(Storage{}, key_set, data_type, std::move(device_opt)) {}
```

Comment:
What was the behavior of this before? :p

Reply:
Same behavior, I needed when I added the new constructor. I can change this back but I do think this makes it more explicit. Maybe there is some not copy, emplace_back thing though performance wise going on by just having the braces

Reply:
I like the added explicitness, just curious if it was silently incorrect before or something

Reply:
I'll change back in abundance of caution